### PR TITLE
Parameterize token info

### PR DESCRIPTION
**Pull request by AnieeG**
### Requires

Needed to parameterize token info through OCRParams instead of hardcoding the values to always use link and weth

### CODE REVIEW

File: deployment/ccip/changeset/initial_add_chain.go [hunk 1]
```diff
@@ -192,7 +191,6 @@ func DefaultOCRParams(
 			RootSnoozeTime:            *config.MustNewDuration(internal.RootSnoozeTime),
 			MessageVisibilityInterval: *config.MustNewDuration(internal.FirstBlockAge),
 			BatchingStrategyID:        internal.BatchingStrategyID,
-			TokenDataObservers:        dataObserverConfig,
```
Comment (AnieeG) [inline deployment/ccip/changeset/initial_add_chain.go:195]: We are deriving this from USDC config in configureChains.

**Review by connorwstein — DISMISSED** [review]

File: deployment/ccip/changeset/internal/deploy_home_chain.go [hunk 2]
```diff
@@ -28,7 +28,7 @@ const (
 
 	FirstBlockAge                           = 8 * time.Hour
 	RemoteGasPriceBatchWriteFrequency       = 30 * time.Minute
-	TokenPriceBatchWriteFrequency           = 3 * time.Second
+	TokenPriceBatchWriteFrequency           = 30 * time.Minute
```
Comment (cmalec) [inline deployment/ccip/changeset/internal/deploy_home_chain.go:31]: this is a big change just want to call it out to make sure its what you intended

**Review by cmalec — DISMISSED** [review]

File: deployment/ccip/changeset/internal/deploy_home_chain.go (hunk 2 above)
Comment (AnieeG) [reply deployment/ccip/changeset/internal/deploy_home_chain.go:31]: I was testing something locally and mistakenly changed it from 30 min to 3s in another PR. This is the intended one. :)

**Review by connorwstein — DISMISSED** [review]

**Review by winder — DISMISSED** [review]

File: deployment/ccip/changeset/test_helpers.go [hunk 3]
```diff
@@ -307,10 +314,8 @@ func NewMemoryEnvironmentWithJobsAndContracts(t *testing.T, lggr logger.Logger,
 			SourceMessageTransmitterAddr: state.Chains[chain].MockUSDCTransmitter.Address().String(),
 		}
 	}
-	for _, chain := range allChains {
-		timelocksPerChain[chain] = state.Chains[chain].Timelock
-		ocrParams[chain] = DefaultOCRParams(e.FeedChainSel, nil, nil)
-	}
+	require.NotNil(t, state.Chains[e.FeedChainSel].LinkToken)
+	require.NotNil(t, state.Chains[e.FeedChainSel].Weth9)
```
Comment (b-gopalswami) [inline deployment/ccip/changeset/test_helpers.go:318]: Do we need this as we are doing similar validation for all the chains?
https://github.com/smartcontractkit/chainlink/pull/15426/files#diff-647d1f54557a9137c5711f752ef36c682ac55c1af4883782fda38622ed6b0a70R367-R379

**Review by b-gopalswami — DISMISSED** [review]

**Review by b-gopalswami — APPROVED** [review]

**Review by winder — APPROVED** [review]

**Review by cmalec — APPROVED** [review]

**Pull request merged**